tests/functional/products/vpn/test_landing.py

```diff
@@ -10,7 +10,41 @@
 @pytest.mark.nondestructive
 @pytest.mark.parametrize(
     "country",
-    [("us"), ("ca"), ("my"), ("nz"), ("sg"), ("gb"), ("de"), ("fr"), ("at"), ("be"), ("ch"), ("es"), ("it"), ("ie"), ("nl"), ("se"), ("fi")],
+    [
```

Comment:
We can't update these tests just yet as the new countries are behind a switch (this is something we usually do post-launch after cleaning up the switches)

Reply:
Ah of course, and it passes locally because dev mode but would fail stage and prod. Good catch.